virttest/utils_test/__init__.py

```diff
@@ -790,7 +795,12 @@ def get_results(base_results_dir):
         compress_cmd += "tar cjvf %s ./*" % results_tarball
         compress_cmd += " --exclude=*core*"
         compress_cmd += " --exclude=*crash*"
-        session.cmd(compress_cmd, timeout=600)
+        while True:
```

Comment:
Agree with @PandaWei 's comments, you should find out root cause of output of extra_cmd is not expect, not repeat, repeat and repeat...